<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NamedField</class>
 <widget class="QWidget" name="NamedField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>154</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NamedField</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="icon">
        <iconset theme="help-about"/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="fieldLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="unitsLabel">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>units</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
